src/docs/user/references/Variables.adoc

```diff
@@ -142,8 +142,11 @@ Examples: PA:JSON will accept {"name": "John", "city":"New York"} and empty valu
 
 ===== Variable Substitution
 Models can use the value of other variables in their definition. +
-For example: +
-The model `PA:LIST($var1,$var2,$var3)` uses the value of the 3 variables `var1`, `var2` and `var3` to build the list of acceptable entries.
+To define a variable substitution, the workflow designer can simply add `$` at the beginning of another variable name. +
```

Comment:
You mean the name of the variable to substitute? If yes, i would specify more.

Reply:
This is done in the example that Medou provides right after this sentence.

Reply:
Indeed. It s just a suggestion to make it more clear (from the beginning).